deploy/helm/elastic-agent/values.yaml

```diff
@@ -0,0 +1,455 @@
+  jobs:
+    state:
+      # -- enable jobs state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_job)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- jobs state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  cronjobs:
+    state:
+      # -- enable cronjobs state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_cronjob)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- cronjobs state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  persistentvolumes:
+    state:
+      # -- enable persistentvolumes state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_persistentvolume)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- persistentvolumes state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  persistentvolumeclaims:
+    state:
+      # -- enable persistentvolumeclaims state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_persistentvolumeclaim)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- persistentvolumeclaims state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  resourcequotas:
+    state:
+      # -- enable resourcequotas state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_resourcequota)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- resourcequotas state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  services:
+    state:
+      # -- enable services state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_service)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- services state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  system:
+    metrics:
+      # -- enable system metric stream (kubelet) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kubelet#system)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- system metric stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+
+# -- extra [user-defined integrations](https://www.elastic.co/guide/en/fleet/current/elastic-agent-input-configuration.html) to be added to the Elastic Agent
+# An example can be found [here](./examples/nginx-custom-integration/README.md)
+# @section -- 3 - User Extra Integrations
+extraIntegrations: {}
+
+agent:
+  # -- enable elastic-agent
+  # @section -- 3 - Elastic-Agent Configuration
+  enabled: true
```

Comment:
`enabled: false` would mean what here?

Reply:
so there was this "notion" back in the day of being able to deploy an elastic-based agent or another agent that the configs are gonna be adjusted to that, hence the `enabled` flag. But you are right, this is not needed and I will remove it

Reply:
I removed this option, please feel free to unresolve if you spot anything else